### PR TITLE
content changes and links restructured

### PR DESCRIPTION
**Pull request by karthick-core**
*Issue #, if available:*

*Description of changes:*
content changes and links restructured

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/update-site.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Update Site
```
Comment (eugenemu) [inline .github/workflows/update-site.yaml]: This doesn't need to be added.

File: content/030_self_guided_setup/32_request_credit.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ weight: 12
 <tr class="credit">
   <td class="credit" style="width:70%">To help you cover the AWS service costs incurred by doing this tutorial,
```
Comment (eugenemu) [inline content/030_self_guided_setup/32_request_credit.md]: Can we rollback changes on this file?  Doesn't seem necessary

File: .github/workflows/update-site.yaml (hunk 1 above)
Comment (karthick-core) [reply .github/workflows/update-site.yaml]: file can be removed/removed in the reference fork.

File: content/030_self_guided_setup/32_request_credit.md (hunk 2 above)
Comment (karthick-core) [reply content/030_self_guided_setup/32_request_credit.md]: removed the lines. should we consider removing the file itself?

**Review by eugenemu — APPROVED** [review]

**Pull request merged**